tests/integrationv2/test_external_psk.py

```diff
@@ -0,0 +1,79 @@
+import copy
+import os
+import pytest
+import time
+
+from configuration import available_ports, TLS13_CIPHERS, ALL_TEST_CURVES, ALL_TEST_CERTS
+from common import ProviderOptions, Protocols, data_bytes, Curves
+from fixtures import managed_process
+from providers import Provider, S2N, OpenSSL
+from utils import invalid_test_parameters, get_parameter_name
+
+CLIENT_PSK_PARAMETERS = [ 
+    ['--psk', 'id,secret,S2N_PSK_HMAC_SHA256'],  # No shared PSK 
+    ['--psk', 'id1,secret1,S2N_PSK_HMAC_SHA224', # Shared PSK at the wire index 1
+     '--psk', 'shared_id,shared_secret,S2N_PSK_HMAC_SHA256'],
```

Comment:
Nit: it took me looking at this several times to realize that there's two PSKs in this list, because I don't expect a mid-list comment :) Can you put the comments ABOVE the lists, instead of in the middle of them?